File: 16x9.py
#!/usr/bin/env python3

for j in [16, 8, 4, 2]:
    print("mod ", j)
    for i in range(1, 1000):
        h = i*j
        if h % 9 == 0:
            print("*", h*16//9, "x", h)

File: 16x9.js
var mods = [16, 8, 4, 2];

for (var j in mods) {
    console.log("mod " + mods[j] + " ");

    for (var i = 1; i < 600; i++) {
        var h = i * mods[j];
        if ((h % 9) === 0) {
            console.log(h * 16 / 9 + "x" + h + " ");
        }
    }
}
